.github/workflows/lint.yml

```diff
@@ -68,27 +68,25 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      - name: Get HEAD commit SHA
-        run: echo ::set-output name=commit-sha::$(git rev-parse HEAD)
-        id: get-commit-sha
+      - name: Prepare output dir with HEAD commit SHA
+        run: |
+          mkdir flake8-output
+          cd flake8-output
+          echo ${{ github.event.pull_request.head.sha }} > commit-sha.txt
       - name: Run flake8
         run: |
           set -eux
           pip install -r requirements-flake8.txt
           flake8 --version
-          flake8 | tee ${GITHUB_WORKSPACE}/flake8-output.txt
-      - name: Add annotations
-        uses: pytorch/add-annotations-github-action@master
+          flake8 | tee ${GITHUB_WORKSPACE}/flake8-output/warnings.txt
+      - name: Upload artifact
```

Comment:
wondering did we measure the impact of these artifacts upload in terms of performance/overall CI time? clang-tidy can get huge

Reply:
oh good question, I actually have no idea